src/components/updatemenus/attributes.js

```diff
@@ -57,6 +57,37 @@ module.exports = {
         ].join(' ')
     },
 
+    openreverse: {
```

Comment:
Default is to the right, if horizontal, or down, if vertical. Hence 'reverse' instead of just `up`/`down`.